### PR TITLE
Added tests for /info endpoint

### PR DESCRIPTION
**Pull request by sam-i-am012**
I didn't realize I forgot to add tests to this endpoint.
Addresses #68

### CODE REVIEW

File: src/main/java/org/example/Operations/InformationOperations.java [hunk 1]
```diff
@@ -18,11 +18,7 @@ public InformationOperations(IrodsHttpClient client) {
         this.client = client;
         this.baseUrl = client.getBaseUrl() + "/info";
     }
-    /**
-     * Sends request to /info endpoint and parses the response
-     * @return Info objected parsed from the response JSON
-     */
-    public Response info() {
+    public Response get() {
```
Comment (korydraughn) [inline src/main/java/org/example/Operations/InformationOperations.java:21]: Add an empty line above this.

**Review by korydraughn — COMMENTED** [review]
Just one comment to resolve.

File: src/main/java/org/example/Operations/InformationOperations.java (hunk 1 above)
Comment (sam-i-am012) [reply src/main/java/org/example/Operations/InformationOperations.java:21]: Done.

**Comment by sam-i-am012** [comment]
Resolved all existing commits, also removed the authentication call to the rods user

**Review by korydraughn — COMMENTED** [review]
If the test still passes, squash it. No pounds.

Let's give the renaming of `.info` to `.get` a real issue number.

The general housekeeping issue should only be used for things like typos, grammar, whitespace corrections, and things that don't change the behavior of the software.

**Comment by sam-i-am012** [comment]
Test still passes. I gave it a real issue number and squashed

**Review by korydraughn — APPROVED** [review]
Pound it.

**Comment by sam-i-am012** [comment]
Done.

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**